code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -150,6 +150,32 @@
 	desc = "A sturdy black military belt. Can hold small weapons, magazines, or smaller boxes of ammunition."
 	icon_state = "militarybelt_black"
 	item_state = "military_black"
+	
+/obj/item/weapon/storage/belt/ninja
+	name = "ninja belt"
+	desc = "A belt designed to hold throwing stars. Not really that good at doing anything else."
+	icon_state = "militarybelt_black"
+	item_state = "military_black"
+	can_hold = list(
+		/obj/item/weapon/throwing_star
+		)
+		
+/obj/item/weapon/storage/belt/ninja/full/New()
+	..()
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
+	new /obj/item/weapon/throwing_star(src)
```

Comment:
Can I suggest using a loop? It would allow you to easily change the number of stars.

As an addition, I'm not sure if possible but you could have a parameter (default value 15) which is how many stars to spawn?